docs/docs-content/release-notes/release-notes.md

```diff
@@ -11,6 +11,178 @@ tags: ["release-notes"]
+  usage between the agent and the Palette management platform.
+
+- Palette API responses now include the header`Cache-Control`. This header provides information on how long the response
+  can be cached and helps improve the performance of the Palette UI.
+
+- Self-Hosted Palette and Private Cloud Gateway (PCG) instances deployed on VMware vSphere now use the vSphere CSI
+  driver version 3.2.0. The new version will automatically get picked up during an upgrade.
+
+### Edge
+
+#### Breaking Changes
+
+- A change in the EdgeForge process affects the Local UI customization process when using the CanvOS utility. In the
+  past, placing a folder named **ui** at the root level of the CanvOS project was required. Moving forward, the **ui**
+  folder will be placed in the **local-ui/** folder. This change is to align with the new CanvOS project structure. If
+  you are using the EdgeForge process to create Edge artifacts, ensure you update the location of the **ui** folder in
+  your CanvOS project. Refer to the Local UI [Custom Links](../clusters/edge/local-ui/host-management/custom-link.md)
+  and [Customize Local UI Theme](../clusters/edge/local-ui/host-management/theming.md) to learn more about the changes.
+
+#### Features
+
+- A new Palette API endpoint, `v1/edgehosts/tags`, is available to retrieve all tags associated with Edge clusters.
+
+- [The Edge Management API](/api/category/edge-management-api-v1/) now supports some operations on connected Edge hosts
+  (non-airgap). In the past, the Edge Management API only supported airgap Edge hosts and clusters. The new
+  functionality now allows you to perform some actions using the Edge Management API on connected Edge hosts. Refer to
+  [List of Endpoints Unavailable to Connected Edge Hosts](/api/introduction/#list-of-endpoints-unavailable-to-connected-edge-hosts)
+  section to learn more about the limitations of connected Edge hosts.
+
+- Local UI now supports signed content bundles and cluster definitions. You can embed a public key in your Edge
+  Installer ISO or provider image. Local UI can use the key to verify the content bundle and cluster definition
+  cryptographically during uploads to ensure you are fulfilling compliance requirements. Refer to the
+  [Build Content Bundles](../clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md) guide to learn
+  more.
+
+#### Improvements
+
+- You can now disable password changes of Operating System (OS) users from Local UI. When password updates from Local UI
+  are disabled, you can still update the OS user password from the OS or the Edge Management API. Check out the
+  [Access Local UI](../clusters/edge/local-ui/host-management/access-console.md) page to learn more.
+
+- Several enhancements have been made to the UI for Edge host management in the context of cluster creation and updates.
+  The new changes ensure a consistent and user-friendly experience that involves new designs for the edge host selection
```

Comment:
```suggestion
  The new changes ensure a consistent and user-friendly experience that involves new designs for the Edge host selection
```

Reply:
This sentence reads slightly awkward. Maybe:

> These changes ensure a consistent and user-friendly experience, including new designs for the Edge host selection screen and a customizable, powerful grid view. This redesign provides a better user experience for managing many edge hosts in large-scale environments. Check out the [Edge Host Grid View](../clusters/edge/site-deployment/edge-host-view.md) page to learn more.

Reply:
Thank you Lenny